library/src/main/java/com/google/android/cameraview/CameraView.java

```diff
@@ -152,6 +153,9 @@ protected void onDetachedFromWindow() {
             mDisplayOrientationDetector.disable();
         }
         super.onDetachedFromWindow();
+        if (mCallbacks != null && mCallbacks.mCallbacks != null) {
+            mCallbacks.mCallbacks.clear();
```

Comment:
Important for memory leak: clear out any callbacks to ensure no extra references are held